control-plane/README.md

```diff
@@ -41,4 +54,4 @@ Created and Updated sorts are much simpler and only require a single query to th
 A user can search for jobs by Job ID. To search for multiple jobs, separate with semicolons.
 
 ## Table
-The table UI is based on [Material Design Lite (MDL)](https://getmdl.io/). The top level of the table contains information about the job. When you click on the job, it will display information about the sub-jobs. 
+The table UI is based on [Material Design Lite (MDL)](https://getmdl.io/). The top level of the table contains information about the job. When you click on the job, it will display information about the sub-jobs. 
```

Comment:
MDL probably uses Apache-2 license. Could you verify that?

Reply:
It does use Apache-2